### PR TITLE
feat(tab): add tab header element

### PR DESCRIPTION
**Pull request by fogfish**
See #406

### CODE REVIEW

**Comment by rbarilani** [comment]
Hi @fogfish good to see your contribution coming....

Can you please:

1. adjust the commit message following our [commit guidelines](https://github.com/zalando/dress-code/blob/master/docs/guides/CONTRIBUTING.md#-git-commit-guidelines).

```
feat(tab): add tab header element

close #422
```

2. Add ```.dc-tab__element--header``` selector  so can be used as css class

```scss

/// ...

@mixin dc-tab-selectors {
    /// ...
    .dc-tab__element {
        @include dc-tab__element;
    }

    .dc-tab__element--header {
        @include  dc-tab__element--header;
    }
/// ...
}
```

Thanks in advance.

**Comment by fogfish** [comment]
Gentlemens, this should be aligned with your requirements.

File: docs/demo/materials/03-atoms/tab.html [hunk 1]
```diff
@@ -8,12 +8,15 @@
 
     `dc-tab__element`: Use this class to create another tab.
 
+    `dc-tab__element--active`: to show a tab as header.
```
Comment (sambhav-gore) [inline docs/demo/materials/03-atoms/tab.html]: this should be `dc-tab__element--header`

Comment (fogfish) [reply docs/demo/materials/03-atoms/tab.html]: Good catch! Now, it is fixed.

**Comment by rbarilani** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Comment by dami-gg** [comment]
Thanks for your contribution @fogfish!

**Comment by dami-gg** [comment]
Moreover, for your information @fogfish we'll be heavily working on Dress Code next week and plan to release a new version which will include your contribution. 

You can follow it here as well: https://github.com/zalando/dress-code/milestone/15

**Pull request merged**